group-coordinator/src/main/java/org/apache/kafka/coordinator/group/assignor/MemberAssignment.java

```diff
@@ -54,6 +60,6 @@ public int hashCode() {
 
     @Override
     public String toString() {
-        return "MemberAssignment(targetPartitions=" + targetPartitions + ')';
+        return "MemberAssignment (Target partitions = " + targetPartitions + ')';
```

Comment:
nit: The format does not follow our convention whereas the former code did. Could we revert this line back?

Reply:
fixed thanks